samples/javascript_nodejs/05.multi-turn-prompt/bot.js

```diff
@@ -165,4 +165,4 @@ class MainDialog {
     }
 }
 
-module.exports = MainDialog;
+module.exports = MultiTurnBot;
```

Comment:
stevenic has suggested that we want to always to named exports.  so module.exports.MultiTurnBot = MultiTurnBot.  He likes this pattern as it forces the naming of the associated require();